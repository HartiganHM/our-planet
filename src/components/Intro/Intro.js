import React from 'react';
import copyContent from '../../data/copyContent';
import './Intro.css';

const Intro = () => {
  return (
    <div className='Intro'>
      <div className='intro-body'>{copyContent.intro}</div>
    </div>
  )
}

export default Intro