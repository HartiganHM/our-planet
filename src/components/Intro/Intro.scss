@import '../../styles/mixins.scss';

.Intro {
  .intro-body {
    @include maxWidth(55);
    margin: auto;
    @include contentDetailBody();
  }
  .intro-logo {

  }
}