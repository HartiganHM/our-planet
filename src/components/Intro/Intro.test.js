import React from 'react';
import { shallow } from 'enzyme';
import Intro from './Intro';

describe('Intro tests', () => {
  it('Should match the snapshot', () => {
    const renderedIntro = shallow(<Intro />);

    expect(renderedIntro).toMatchSnapshot();
  });
});
