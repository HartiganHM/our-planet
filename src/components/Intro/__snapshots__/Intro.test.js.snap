// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro tests Should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Intro />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="intro-body"
>
          <p>
                    Our Planet is a student lead initiative meant to bring awareness to endangered animals across the world through detailed facts and beautiful photographs. While most strive to be aware of other people in need, it is less frequent that we pay mind to the other species that cohabitate this world with us.
          </p>
          <p>
                    This site puts a face to those animals with the greatest need, in hope that their plight resonates with someone somewhere. When you have time to unplug and breath fresh air, remember that Earth isn't your planet or my planet, but...
          </p>
</div>,
        <div
          className="intro-logo"
/>,
      ],
      "className": "Intro",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
              Our Planet is a student lead initiative meant to bring awareness to endangered animals across the world through detailed facts and beautiful photographs. While most strive to be aware of other people in need, it is less frequent that we pay mind to the other species that cohabitate this world with us.
</p>,
            <p>
              This site puts a face to those animals with the greatest need, in hope that their plight resonates with someone somewhere. When you have time to unplug and breath fresh air, remember that Earth isn't your planet or my planet, but...
</p>,
          ],
          "className": "intro-body",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Our Planet is a student lead initiative meant to bring awareness to endangered animals across the world through detailed facts and beautiful photographs. While most strive to be aware of other people in need, it is less frequent that we pay mind to the other species that cohabitate this world with us.",
            },
            "ref": null,
            "rendered": "Our Planet is a student lead initiative meant to bring awareness to endangered animals across the world through detailed facts and beautiful photographs. While most strive to be aware of other people in need, it is less frequent that we pay mind to the other species that cohabitate this world with us.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "This site puts a face to those animals with the greatest need, in hope that their plight resonates with someone somewhere. When you have time to unplug and breath fresh air, remember that Earth isn't your planet or my planet, but...",
            },
            "ref": null,
            "rendered": "This site puts a face to those animals with the greatest need, in hope that their plight resonates with someone somewhere. When you have time to unplug and breath fresh air, remember that Earth isn't your planet or my planet, but...",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "intro-logo",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="intro-body"
>
            <p>
                        Our Planet is a student lead initiative meant to bring awareness to endangered animals across the world through detailed facts and beautiful photographs. While most strive to be aware of other people in need, it is less frequent that we pay mind to the other species that cohabitate this world with us.
            </p>
            <p>
                        This site puts a face to those animals with the greatest need, in hope that their plight resonates with someone somewhere. When you have time to unplug and breath fresh air, remember that Earth isn't your planet or my planet, but...
            </p>
</div>,
          <div
            className="intro-logo"
/>,
        ],
        "className": "Intro",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                Our Planet is a student lead initiative meant to bring awareness to endangered animals across the world through detailed facts and beautiful photographs. While most strive to be aware of other people in need, it is less frequent that we pay mind to the other species that cohabitate this world with us.
</p>,
              <p>
                This site puts a face to those animals with the greatest need, in hope that their plight resonates with someone somewhere. When you have time to unplug and breath fresh air, remember that Earth isn't your planet or my planet, but...
</p>,
            ],
            "className": "intro-body",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Our Planet is a student lead initiative meant to bring awareness to endangered animals across the world through detailed facts and beautiful photographs. While most strive to be aware of other people in need, it is less frequent that we pay mind to the other species that cohabitate this world with us.",
              },
              "ref": null,
              "rendered": "Our Planet is a student lead initiative meant to bring awareness to endangered animals across the world through detailed facts and beautiful photographs. While most strive to be aware of other people in need, it is less frequent that we pay mind to the other species that cohabitate this world with us.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "This site puts a face to those animals with the greatest need, in hope that their plight resonates with someone somewhere. When you have time to unplug and breath fresh air, remember that Earth isn't your planet or my planet, but...",
              },
              "ref": null,
              "rendered": "This site puts a face to those animals with the greatest need, in hope that their plight resonates with someone somewhere. When you have time to unplug and breath fresh air, remember that Earth isn't your planet or my planet, but...",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "intro-logo",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
